cmd/webhook/app/options/options.go

```diff
@@ -58,7 +67,16 @@ func (o *WebhookOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.DynamicServingCASecretName, "dynamic-serving-ca-secret-name", "", "name of the secret used to store the CA that signs serving certificates certificates")
 	fs.StringSliceVar(&o.DynamicServingDNSNames, "dynamic-serving-dns-names", []string{""}, "DNS names that should be present on certificates generated by the dynamic serving CA")
 	fs.StringVar(&o.Kubeconfig, "kubeconfig", "", "optional path to the kubeconfig used to connect to the apiserver. If not specified, in-cluster-config will be used")
-	fs.StringSliceVar(&o.TLSCipherSuites, "tls-cipher-suites", defaultCipherSuites, "comma separated list of TLS 1.2 cipher suites to use (TLS 1.3 cipher suites are not configurable)")
+
+	tlsCipherPossibleValues := cliflag.TLSCipherPossibleValues()
+	fs.StringSliceVar(&o.TLSCipherSuites, "tls-cipher-suites", o.TLSCipherSuites,
+		"Comma-separated list of cipher suites for the server. "+
+			"If omitted, the default Go cipher suites will be use.  "+
+			"Possible values: "+strings.Join(tlsCipherPossibleValues, ","))
+	tlsPossibleVersions := cliflag.TLSPossibleVersions()
+	fs.StringVar(&o.MinTLSVersion, "tls-min-version", o.MinTLSVersion,
+		"Minimum TLS version supported. "+
+			"Possible values: "+strings.Join(tlsPossibleVersions, ", "))
```

Comment:
It's not clear to me what the default will be. 
Previously it is being set to VersionTLS12,
now it will be set to empty string, AFAICS.
....ah, which is then validated by https://github.com/kubernetes/component-base/blob/master/cli/flag/ciphersuites_flag.go#L91 which returns a Kubernetes Default version if the string is empty.

I see. Maybe add a comment to the MinTLSVersion explaining where the default comes from.
Or maybe supply `flag.DefaultVersion()` here, so that the CLI --help will show what the default version is.

Reply:
I can add a comment/set defaults 😄 I'm not actually too sure why other things k8s components don't set this default properly either..